source/agora/test/DNS.d

```diff
@@ -157,4 +157,7 @@ unittest
     // CNAME record
     result = resolver.query(node_addr ~ ".validators.unittest.bosagora.io.", QTYPE.CNAME);
     assert(result.length == 0);
+
+    result = resolver.query("validators.unittest.bosagora.io.", QTYPE.AXFR);
+    assert(result is null);
```

Comment:
How come there's no result?

Reply:
Resolver sets return value to null when RCode is different than NoError. Caching registry doesn’t have AXFR capabilities.